CommonLibSF/include/RE/T/TESObjectREFR.h

```diff
@@ -285,14 +285,18 @@ namespace RE
 
 		// NiPointer<TESObjectREFR>
 		[[nodiscard]] TESObjectREFR* GetAttachedSpaceship();
-		[[nodiscard]] TESWorldSpace* GetParentWorldSpace();
+
```

Comment:
Remove the extra 2 empty lines you added here (Might not be needed if clang-format fixes that, not sure)